streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamFlatTransformTest.java

```diff
@@ -47,19 +56,14 @@ public class KStreamFlatTransformTest extends EasyMockSupport {
     public void setUp() {
         inputKey = 1;
         inputValue = 10;
-        transformer = mock(Transformer.class);
-        context = strictMock(InternalProcessorContext.class);
```

Comment:
In EasyMock a strict mock verifies also the order of the calls. In Mockito that is done by using `InOrder` (https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#in_order_verification). Currently, this PR does not check the order which softens up the test.

Reply:
Addressed in subsequent commits.